br.com.gualandi.Xjump.yml

```diff
@@ -0,0 +1,20 @@
+app-id: br.com.gualandi.Xjump
```

Comment:
Is this domain owned by the developer? it seems to redirect to something else.

Reply:
I do own the domain. I concede that the redirect is confusing though.